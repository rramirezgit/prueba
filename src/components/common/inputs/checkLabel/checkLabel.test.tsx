import React from 'react'
import renderer from 'react-test-renderer'
import CheckLabel from '.'

test('renders CheckLabel correctly', () => {
  const tree = renderer.create(<CheckLabel label="Remember me" />).toJSON()
  expect(tree).toMatchSnapshot()
})
