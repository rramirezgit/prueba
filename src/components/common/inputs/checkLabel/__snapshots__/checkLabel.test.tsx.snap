// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CheckLabel correctly 1`] = `
<div
  className="rememberMe"
>
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-dx0q9x-MuiCheckbox-root{color:rgba(0, 0, 0, 0.6);padding:0px;margin:0px 10px 0px 0px;}.css-dx0q9x-MuiCheckbox-root:hover{background-color:rgba(25, 118, 210, 0.04);}@media (hover: none){.css-dx0q9x-MuiCheckbox-root:hover{background-color:transparent;}}.css-dx0q9x-MuiCheckbox-root.Mui-checked,.css-dx0q9x-MuiCheckbox-root.MuiCheckbox-indeterminate{color:#1976d2;}.css-dx0q9x-MuiCheckbox-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);}",
      }
    }
    data-emotion="css dx0q9x-MuiCheckbox-root"
  />
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-1bh5t8v-MuiCheckbox-root{padding:9px;border-radius:50%;color:rgba(0, 0, 0, 0.6);padding:0px;margin:0px 10px 0px 0px;}.css-1bh5t8v-MuiCheckbox-root:hover{background-color:rgba(25, 118, 210, 0.04);}@media (hover: none){.css-1bh5t8v-MuiCheckbox-root:hover{background-color:transparent;}}.css-1bh5t8v-MuiCheckbox-root.Mui-checked,.css-1bh5t8v-MuiCheckbox-root.MuiCheckbox-indeterminate{color:#1976d2;}.css-1bh5t8v-MuiCheckbox-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);}",
      }
    }
    data-emotion="css 1bh5t8v-MuiCheckbox-root"
  />
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-17i0tha-MuiButtonBase-root-MuiCheckbox-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;position:relative;box-sizing:border-box;-webkit-tap-highlight-color:transparent;background-color:transparent;outline:0;border:0;margin:0;border-radius:0;padding:0;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;vertical-align:middle;-moz-appearance:none;-webkit-appearance:none;-webkit-text-decoration:none;text-decoration:none;color:inherit;padding:9px;border-radius:50%;color:rgba(0, 0, 0, 0.6);padding:0px;margin:0px 10px 0px 0px;}.css-17i0tha-MuiButtonBase-root-MuiCheckbox-root::-moz-focus-inner{border-style:none;}.css-17i0tha-MuiButtonBase-root-MuiCheckbox-root.Mui-disabled{pointer-events:none;cursor:default;}@media print{.css-17i0tha-MuiButtonBase-root-MuiCheckbox-root{-webkit-print-color-adjust:exact;color-adjust:exact;}}.css-17i0tha-MuiButtonBase-root-MuiCheckbox-root:hover{background-color:rgba(25, 118, 210, 0.04);}@media (hover: none){.css-17i0tha-MuiButtonBase-root-MuiCheckbox-root:hover{background-color:transparent;}}.css-17i0tha-MuiButtonBase-root-MuiCheckbox-root.Mui-checked,.css-17i0tha-MuiButtonBase-root-MuiCheckbox-root.MuiCheckbox-indeterminate{color:#1976d2;}.css-17i0tha-MuiButtonBase-root-MuiCheckbox-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);}",
      }
    }
    data-emotion="css 17i0tha-MuiButtonBase-root-MuiCheckbox-root"
  />
  <span
    className="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-root MuiCheckbox-colorPrimary css-17i0tha-MuiButtonBase-root-MuiCheckbox-root"
    onBlur={[Function]}
    onContextMenu={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={null}
  >
    <style
      dangerouslySetInnerHTML={
        {
          "__html": ".css-1m9pwf3{cursor:inherit;position:absolute;opacity:0;width:100%;height:100%;top:0;left:0;margin:0;padding:0;z-index:1;}",
        }
      }
      data-emotion="css 1m9pwf3"
    />
    <input
      className="PrivateSwitchBase-input css-1m9pwf3"
      data-indeterminate={false}
      onChange={[Function]}
      required={false}
      type="checkbox"
    />
    <style
      dangerouslySetInnerHTML={
        {
          "__html": ".css-i4bv87-MuiSvgIcon-root{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:1.5rem;}",
        }
      }
      data-emotion="css i4bv87-MuiSvgIcon-root"
    />
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
      data-testid="CheckBoxOutlineBlankIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
      />
    </svg>
  </span>
  Remember me
</div>
`;
